src/App.jsx

```diff
@@ -14,57 +15,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const DEFAULT_SELECTED = 'Jam';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(DEFAULT_SELECTED);
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const NO_SELECTED_GOOD = '';
+  const isSelectedGood = selectedGood !== NO_SELECTED_GOOD;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const handleClear = () => {
+    setSelectedGood('');
```

Comment:
Since you've defined empty value in `NO_SELECTED_GOOD` constant, it makes sense to use it:  `setSelectedGood(NO_SELECTED_GOOD);`.